demos/src/package-special-report.mustache

```diff
@@ -0,0 +1,28 @@
+<div class="o-topper o-topper--split-text-left o-topper--package o-topper--special-report o-topper--color-claret">
+	<div class="o-topper__content">
+		<div class="o-topper__tags">
+			<a href="https://www.ft.com/reports" class="o-topper__topic tags--large">Special Report</a>
+		</div>
+		<h1 class="o-topper__headline o-topper__headline--large" data-trackable="header">
+			<span class="article-classifier__gap">Europe’s Leading Patent Law Firms</span>
+		</h1>
+		<div class="o-topper__summary o-topper__summary--body">
+			<p>We highlight Europe’s leading patent law firms for 2020. Plus: invention still needs the human touch; pandemic reignites debate on rights protection; UK and Germany hinder Unified Patent Court launch; counterfeiting still rife beyond EU; university spinouts seek full value in commercial applications</p>
+			<p><a href="https://www.statista.com/page/europes-leading-patent-law-firms-2021-registration"><em>Apply for the 2021 ranking here</em></a></p>
+		</div>
+	</div>
+
+	<div class="o-topper__background"></div>
+
+	<figure class="o-topper__visual">
+		<picture class="o-topper__picture">
+			<source media="(min-width: 1440px)" srcset="https://www.ft.com/__origami/service/image/v2/images/raw/https://d1e00ek4ebabms.cloudfront.net/production/0eac32d0-1cec-4f20-a183-d9f4370309fb.jpg?source=next&amp;fit=scale-down&amp;quality=highest&amp;width=1067 1067w">
```

Comment:
A small change can be made to these images to make them use the ftcms scheme. The ftcms scheme is a way to future proof the image as it enables UPP to change where images are stored/served from without having to update the urls in our html
```suggestion
			<source media="(min-width: 1440px)" srcset="https://www.ft.com/__origami/service/image/v2/images/raw/ftcms:0eac32d0-1cec-4f20-a183-d9f4370309fb?source=next&amp;fit=scale-down&amp;quality=highest&amp;width=1067 1067w">
```